hostapd-2.10-wpe.patch not installing (make install) on Raspbian Buster

# Defect Report

## Issue type

<!--
Please select from one of the following software defect descriptions. Pick the one that best reflects the issue you are experiencing.
-->

- [x] Defect - Compilation or `make check` issue - Attach `config.log` and provide relevant system information such as `lscpu` - Make sure dependencies are installed
- [ ] Defect - Crash
- [ ] Defect - Incorrect value displayed/received/stored
- [ ] Defect - Unexpected behavior (obvious or confirmed in the forum)

## System information

- OS Name: Raspbian
- OS Version: Buster
- CPU: armv7I
- Wireless card and chipset (if relevant): N/A

## Aircrack-ng version

- Version: 2.10
- Commit Revision hash: 

## Defect

### Details
Issue is that `sudo make install` fails after you apply the wpe patch on version 2.10:

```
install -D hostapd-wpe /usr/local/bin//hostapd-wpe
install: cannot stat 'hostapd-wpe': No such file or directory
make: *** [Makefile:1277: /usr/local/bin//hostapd-wpe] Error 1
```

To reproduce the issue on raspberry pi 4 with Raspbian Buster:
```
wget https://raw.githubusercontent.com/aircrack-ng/aircrack-ng/master/patches/wpe/hostapd-wpe/hostapd-2.10-wpe.patch
wget https://w1.fi/releases/hostapd-2.10.tar.gz
tar -zxf hostapd-2.10.tar.gz
cd hostapd-2.10
patch -p1 < ../hostapd-2.10-wpe.patch
cd hostapd
```

Then:

```
make
sudo make install
```

Error is:
```
LD  hostapd-wpe
install -D hostapd-wpe /usr/local/bin//hostapd-wpe
install: cannot stat 'hostapd-wpe': No such file or directory
make: *** [Makefile:1277: /usr/local/bin//hostapd-wpe] Error 1
```

Edit by Mister-X-: added backticks for code

--- a/patches/wpe/hostapd-wpe/hostapd-2.10-wpe.patch
+++ b/patches/wpe/hostapd-wpe/hostapd-2.10-wpe.patch
@@ -453,7 +453,7 @@ diff '--color=auto' -rupN hostapd-2.10/hostapd/Makefile hostapd-2.10-wpe/hostapd
  
 -hostapd: $(OBJS)
 +hostapd-wpe: $(OBJS)
- 	$(Q)$(CC) $(LDFLAGS) -o hostapd $(OBJS) $(LIBS)
+ 	$(Q)$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LIBS)
  	@$(E) "  LD " $@
  
 @@ -1291,7 +1301,7 @@ endif
@@ -462,7 +462,7 @@ diff '--color=auto' -rupN hostapd-2.10/hostapd/Makefile hostapd-2.10-wpe/hostapd
  
 -hostapd_cli: $(OBJS_c)
 +hostapd_cli-wpe: $(OBJS_c)
- 	$(Q)$(CC) $(LDFLAGS) -o hostapd_cli $(OBJS_c) $(LIBS_c)
+ 	$(Q)$(CC) $(LDFLAGS) -o $@ $(OBJS_c) $(LIBS_c)
  	@$(E) "  LD " $@
  
 diff '--color=auto' -rupN hostapd-2.10/hostapd/certs/Makefile hostapd-2.10-wpe/hostapd/certs/Makefile

